pandas/core/strings.py

```diff
@@ -27,19 +27,19 @@ def _get_array_list(arr, others):
 
 def str_cat(arr, others=None, sep=None, na_rep=None):
     """
-    Concatenate arrays of strings with given separator
+    Concatenate strings in the Series with given separator
 
     Parameters
     ----------
-    arr : list or array-like
     others : list or array, or list of arrays
+      If None, returns str concatenating strings of the Series
```

Comment:
Ah, but this is also the case .. 
This is quite a different behaviour when having others or not. Maybe some more explanation or an example would be useful.